src/pencilkit.cs

```diff
@@ -483,6 +532,11 @@ interface PKStrokePoint : NSCopying {
 		[DesignatedInitializer]
 		NativeHandle Constructor (CGPoint location, double timeOffset, CGSize size, nfloat opacity, nfloat force, nfloat azimuth, nfloat altitude);
 
+		[Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Export ("initWithLocation:timeOffset:size:opacity:force:azimuth:altitude:secondaryScale:")]
+		[DesignatedInitializer]
```

Comment:
```suggestion
		[DesignatedInitializer]
```
Can we have two designated Initializers in the same interface?

Reply:
yep